src/snowflake/snowpark/modin/plugin/extensions/groupby_overrides.py

```diff
@@ -740,9 +740,59 @@ def ohlc(self):
         ErrorMessage.method_not_implemented_error(name="ohlc", class_="GroupBy")
 
     @_inherit_docstrings(pandas.core.groupby.DataFrameGroupBy.pct_change)
-    def pct_change(self, *args, **kwargs):
+    def pct_change(
+        self,
+        periods=1,
+        fill_method=no_default,
+        limit=no_default,
+        freq=no_default,
+        axis=no_default,
+        **kwargs,
+    ):
         # TODO: SNOW-1063349: Modin upgrade - modin.pandas.groupby.DataFrameGroupBy functions
-        ErrorMessage.method_not_implemented_error(name="pct_change", class_="GroupBy")
+        if fill_method not in (no_default, None) or limit is not no_default:
```

Comment:
why not separate this into two different warnings? It would make the error message easier to read and less verbose.